tests/test_serializers.py

```diff
@@ -1,190 +1,223 @@
 import json
 
-from mock import Mock
+from mock import call, Mock, patch
 
 from tests import SerializerTestCase, TestPreference, TestUser
-from madprops.serializers import PropertySerializer
+from madprops.serializers import PropertySerializer, PropertiesOwnerSerializer
```

Comment:
I101 Imported names are in the wrong order. Should be PropertiesOwnerSerializer, PropertySerializer